### PR TITLE
Update Protobuf Pkg

### PR DESCRIPTION
**Pull request by VladislavsPerkanuks**
https://pkg.go.dev/vuln/GO-2024-2611

### CODE REVIEW

File: pkg/pb/translate/v1/translate.pb.go [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.32.0
+// 	protoc-gen-go v1.33.0
```
Comment (janishorsts) [inline pkg/pb/translate/v1/translate.pb.go:3]: @VladislavsPerkanuks it is a generated file, you know. :)

Comment (VladislavsPerkanuks) [reply pkg/pb/translate/v1/translate.pb.go:3]: it was changed by `earthly +proto`

Comment (VladislavsPerkanuks) [reply pkg/pb/translate/v1/translate.pb.go:3]: It is possible, do you want me to change to specific version ?

Comment (VladislavsPerkanuks) [reply pkg/pb/translate/v1/translate.pb.go:3]: okay, done it, nevertheless :)

Comment (janishorsts) [reply pkg/pb/translate/v1/translate.pb.go:3]: Perfect.

**Review by kaaposc — APPROVED** [review]

**Review by janishorsts — APPROVED** [review]

**Pull request merged**